services/user.py

```diff
@@ -0,0 +1,59 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> None:
+    user = get_user_model().objects.create_user(
+        username=username,
+        password=password,
+    )
+
+    if email:
+        user.email = email
+
+    if first_name:
+        user.first_name = first_name
+
+    if last_name:
+        user.last_name = last_name
```

Comment:
You can replace it with **kwargs, and then unpack kwargs into create_user, so if there was email, first or last names, it will create with correct value

Reply:
Done